ui-tests/src/test/resources/features/integrations/api-provider.feature

```diff
@@ -76,7 +76,7 @@ Feature: API Provider Integration
     Then check visibility of page "Upload API Provider Specification"
 
     # hacky way to reuse the existing step
-    When create API Provider spec from scratch .
+    When create API Provider spec from scratch
```

Comment:
Will this work with the newly refactored `createFromScratch` method? Looks like should specify the openapi version here.

Reply:
If url is null, the default => 2.0 version will be used, but I won't be lazy anymore and change it so it's obvious